common/sgx/tcbinfo.c

```diff
@@ -749,8 +753,8 @@ oe_result_t oe_parse_tcb_info_json(
                 "Found %d AdvisoryIDs for this tcb level.",
                 platform_tcb_level->advisory_ids_size);
         }
-
-        result = OE_OK;
+        if (result != OE_TCB_LEVEL_INVALID)
```

Comment:
This could be a pitfall in the future when other error cases are added. Suggest pre-setting result to OE_OK near line 732 and do nothing here.